src/components/HistoricRatesBody.js

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+import { View, FlatList, ActivityIndicator } from 'react-native';
+import { HistoricRatesStyles as styles } from '../styles';
+import { HistoricRatesItem } from '.';
+import { Query } from 'react-apollo';
+import { GET_COMPUTED_RATES } from '../operations';
+import { sortRatesByDate } from '../dataApi';
+import Whoops from './Whoops';
+
+export default class HistoricRatesBody extends PureComponent {
+
+  render() {
+    return (
+      <Query query={GET_COMPUTED_RATES}>
+        {({ data, loading, error }) => {
+          if (error) {
+            return <Whoops message="Error while fetching rates" />;
+          } else if (loading) {
```

Comment:
Why an else-if ?

Reply:
Lol. Just for the fun of it. An if works naturally, an `else if` can be used once an `if` has already been used